src/components/auth/Login.jsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable react/prop-types */
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link, useNavigate } from 'react-router-dom';
+import { logInUser } from '../../redux/userSlice';
+import './Login.css';
+
+function Login() {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const goToDestination = () => {
+    navigate('/home')
+  }
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  // useEffect(() => {
+  //   hideSplash();
+  // }, [hideSplash]);
+
+  // const isLoading = useSelector((state) => state.user.isLoading);
+  const isAuthenticated = useSelector((state) => state.user.isAuthenticated)
+  // const error = useSelector((state) => state.user.error);
```

Comment:
- [ ] Kindly, remove the old commented code present in this file, in order to improve the repo's cleanliness. 👍